projects/pastanaga/src/lib/controls/filtered-checkbox-group/filtered-checkbox-group.component.scss

```diff
@@ -0,0 +1,86 @@
+@import "../../styles/variables";
+
+:host {
+    position: relative;
+    display: block;
+
+    legend {
+        pa-badge {
+            display: inline-block;
+            position: relative;
+            top: rythm(.25);
+            left: rythm(1.5);
+        }
+        & > span {
+            display: inline-block;
+            margin-top: rythm(.25) * 3;
+        }
+    }
+
+    .filter-container {
+        padding-top: rythm(.5);
+        pa-icon {
+            background-color: rgba($blue, .06);
+            display: block;
+            height: rythm(4);
+            line-height: rythm(4);
+            float: left;
+            border-top-left-radius: rythm(.25);
+            border-bottom-left-radius: rythm(.25);
+            ::ng-deep svg {
+                height: rythm(2);
+                width: rythm(2);
+                fill: $gray-pale;
+                margin-left: rythm(1);
+                position: relative;
+                top: rythm(.5);
+            }
+        }
+
+        input {
+            background-color: rgba($blue, .06);
+            display: inline-block;
+            border: none;
+            border-top-right-radius: rythm(.25);
+            border-bottom-right-radius: rythm(.25);
+            caret-color: $blue;
+            padding: rythm(.25) * 3;
+            padding-left: rythm(1);
+            width: calc(100% - #{rythm(3)});
+            height: rythm(4);
+            line-height: rythm(4);
+            -webkit-box-sizing: border-box;
+            box-sizing: border-box;
+            margin-bottom: rythm(.25);
+    
+            &:hover, &:focus {
+                background-color: rgba($blue, .06);
+    
+                & + pa-icon {
+                    background-color: rgba($blue, .06);
+                }
+            }
+    
+            &:active, &:focus {
+                outline: 0;
+                background-color: rgba($blue, .12);
+    
+                & + pa-icon {
+                    background-color: rgba($blue, .12);
+                }
+            }
+        }
+    }
+
```

Comment:
I don't think that would be needed if you used `pa-input` instead of basic html one.

Reply:
overriding the pa-input style happened to be too difficult here